hkmc2/shared/src/test/mlscript/ucs/general/LogicalConnectives.mls

```diff
@@ -25,19 +25,31 @@ fun test(x) =
 :sjs
 true and test(42)
 //│ JS (unsanitized):
-//│ let scrut6, scrut7;
-//│ scrut6 = true;
-//│ if (scrut6 === true) {
-//│   scrut7 = test(42);
-//│   if (scrut7 === true) { true } else { false }
-//│ } else { false }
+//│ let scrut6, scrut7, tmp4;
+//│ split_root$3: {
+//│   split_1$3: {
+//│     scrut7 = true;
+//│     if (scrut7 === true) {
+//│       scrut6 = test(42);
+//│       if (scrut6 === true) {
+//│         tmp4 = true;
+//│         break split_root$3
+//│       } else {
+//│         break split_1$3
+//│       }
+//│     } else { break split_1$3 }
+//│   }
+//│   tmp4 = false;
```

Comment:
As discussed, it would be good to avoid the labels and breaks when the default branch is small (esp. when it's a simple value).

Reply:
Good suggestion. I’ll add it into the backlog.